cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -6106,24 +6106,25 @@ alert.track = 0
 cron_schedule = 10 * * * *
 description = This report update the device inventory every hour and generates lookup for crowdstrike data. \
 \
-Data Collection - The crowdstrike data needs to be collected via the CrowdStrike Event Stream Add-on (https://splunkbase.splunk.com/app/5082/).
+Data Collection - The crowdstrike data needs to be collected via the CrowdStrike Falcon Devices Technical Add-On (https://splunkbase.splunk.com/app/5570).
 dispatch.earliest_time = -62m@m
 dispatch.latest_time = -2m@m
 display.general.type = statistics
 display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_crowdstrike_eventstream` | eval crowdstrike_userid=coalesce('event.UserId', 'event.UserName', user) | dedup crowdstrike_userid \
-| eval ip=lower(coalesce('event.UserIp', 'event.LocalIP', src)), hostname=lower(coalesce('event.ComputerName', 'event.HostnameField')), time=_time, indextime=_indextime, product_name="CrowdStrike", product_uuid=crowdstrike_userid \
-| rename host as crowdstrike_collected_by, "metadata.customerIDString" as crowdstrike_customer_id, "event.MACAddress" as mac_address \
+search = `cs_crowdstrike_devices` | dedup falcon_device.device_id \
+| eval time=_time, indextime=_indextime, product_name="CrowdStrike Devices" \
+| rename falcon_device.* as * \
+| rename device_id as product_uuid, local_ip as ip, host as crowdstrike_collected_by \
 | fillnull ip, hostname, mac_address, user value="" \
-| table time, indextime, product_name, product_uuid, crowdstrike_collected_by, crowdstrike_customer_id, ip, hostname, mac_address, user, crowdstrike_userid \
+| table _time, time, indextime,  product_name, product_uuid, hostname, user, ip, mac_address, crowdstrike_collected_by, external_ip, agent_version, site_name, status, first_seen, last_seen, last_login_timestamp, last_login_user, last_reboot, os_product_name, os_version, os_build, platform_name, product_type_desc \
 | cyencesdevicemanager operation="addentries" \
-| append [| inputlookup cs_crowdstrike_inventory ] \
-| stats latest(time) as time, latest(*) as * by crowdstrike_userid \
-| eval _key=crowdstrike_userid | outputlookup cs_crowdstrike_inventory
-action.cyences_notable_event_action.products = CrowdStrike EventStream
+| append [| inputlookup cs_crowdstrike_inventory] \
+| dedup product_uuid sortby -_time \
+| eval _key=product_uuid | outputlookup cs_crowdstrike_inventory
```

Comment:
we have to keep crowdstrike_userid field as it is displayed in device inventory raw expansion search.